.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+  - '5.4'
+  - '5.5'
+  - '5.6'
+  - '7.0'
+
+before_script:
+  - git clone https://github.com/glpi-project/glpi -b 9.1 ../glpi && cd ../glpi
+  - mv ../formcreator plugins/formcreator
+  - composer install --no-dev
+  - php tools/cliinstall.php --db=glpi-test --user=travis --tests
```

Comment:
There is no `--db` option, nor `--user` (looks like it is named `--as-user`).

I guess this should be added, along with a `--pass` one that may be needed on local tests; and that would make script compliant with GLPi' one.